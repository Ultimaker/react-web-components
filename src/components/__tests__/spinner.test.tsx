// Copyright (c) 2018 Ultimaker B.V.
import 'jest';
import 'jsdom-global/register';
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import Spinner from '../spinner';

describe('The Spinner component', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Spinner />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });

});
