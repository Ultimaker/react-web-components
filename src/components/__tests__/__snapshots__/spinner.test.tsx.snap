// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Spinner component should render 1`] = `
<div
  className="spinner"
>
  <svg
    height="65px"
    viewBox="0 0 66 66"
    width="65px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="path"
      cx="33"
      cy="33"
      fill="none"
      r="30"
      strokeLinecap="round"
      strokeWidth="6"
    />
  </svg>
</div>
`;
